### DISCUSSION TITLE
All typing lost

### DISCUSSION THREAD

**Discussion by brindis15** [community/community#113247] [New to GitHub]
I cannot believe that github does not have a way to recover uncommited writing (draft). I spend about one hour writing the README file, never commited and the I went back in Chrome... when warning showed up, I cancelled but it went back anyway and I lost my work. I am quite angry at this, now I know that I must commit often, but why don't the add some auto-save feature???

**Comment by queenofcorgis** [comment]
Thanks for the feedback - we appreciate you taking the time to share that with us @brindis15 ❤️
